packages/blockfrost/test/blockfrostProvider.test.ts

```diff
@@ -118,96 +136,95 @@ describe('blockfrostProvider', () => {
     });
   });
 
-  test('utxoDelegationAndRewards', async () => {
-    const addressesUtxosAllMockResponse = [
-      {
-        amount: [
-          {
-            quantity: '50928877',
-            unit: 'lovelace'
-          },
-          {
-            quantity: '1',
-            unit: 'b01fb3b8c3dd6b3705a5dc8bcd5a70759f70ad5d97a72005caeac3c652657675746f31333237'
-          }
-        ],
-        block: 'b1b23210b9de8f3edef233f21f7d6e1fb93fe124ba126ba924edec3043e75b46',
-        output_index: 0,
-        tx_hash: '0f3abbc8fc19c2e61bab6059bf8a466e6e754833a08a62a6c56fe0e78f19d9d5',
-        tx_index: 0
-      },
-      {
-        amount: [
-          {
-            quantity: '1097647',
-            unit: 'lovelace'
-          }
+  describe('utxoDelegationAndRewards', () => {
```

Comment:
I think it would be cleaner to test`fetchByAddressSequentially` separately and then mock it for all Blockfrost requests that uses it. Not a requested change, just a note.

Reply:
Good point. I'll make a ticket to follow-up